packages/auth-helpers/src/lib/recap/utils.ts

```diff
@@ -0,0 +1,57 @@
+import { LitAbility, LitResourcePrefix } from '../models';
+
+/**
+ * Map from a LitAbility to the Recap namespace and ability.
+ * @throws Error if the LitAbility is unknown
+ */
+export function getRecapNamespaceAndAbility(litAbility: LitAbility): {
+  recapNamespace: string;
+  recapAbility: string;
```

Comment:
since `recapNamespace` and `recapAbility` only have limited options defined in enum, we can just do:

```ts
recapNamespace: LitNamespace;
recapAbility: LitRecapAbility;
```